process/smartContract/process.go

```diff
@@ -216,9 +216,16 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return err
 	}
 
+	var txHash []byte
+	txHash, err = core.CalculateHash(sc.marshalizer, sc.hasher, tx)
```

Comment:
Optional - add TODO to get the hash from `TransactionWrapper` (or `TransactionHandler.GetHash()` - in the future, when available).

Optional - why not using `computeTransactionHash()` already defined?

Reply:
discussed with Iulian - we can have a private field in the transaction / scr and other structures with Set and GetHash, not possible right now because of the way proto is integrated.